src/main/scala/firrtl/backends/verilog/VerilogEmitter.scala

```diff
@@ -1022,71 +1022,105 @@ class VerilogEmitter extends SeqTransform with Emitter {
           }
           instdeclares += Seq(");")
         case sx: DefMemory =>
+          require(sx.writeLatency == 1)
+          require(sx.readLatency <= 1)
```

Comment:
```suggestion
```
I'd delete these since they conflict with the exception thrown below. That or the error message in the exception should be migrated here.

Reply:
Done

Reply:
Done